packages/app-bridge/src/AppBridgePlatformApp.spec.ts

```diff
@@ -64,4 +64,24 @@ describe('AppBridgePlatformApp', () => {
         const state = platformApp.state();
         expect(state).toEqual(undefined);
     });
+    it('should yield true for Context.connected after dispatch', async () => {
+        const connected = true;
+        window.location.search = `?token=${TOKEN}`;
+        const platformApp = new AppBridgePlatformApp();
+        platformApp.context().subscribe((context) => {
+            expect({ parentId: 'parentId-test' }).toStrictEqual(context);
+        });
+        platformApp.context('parentId').subscribe((parentId) => {
+            expect('parentId-test').toStrictEqual(parentId);
+        });
+        platformApp.subscribe('Context.connected', () => {
+            const context = platformApp.context().get();
+            const parentId = platformApp.context('parentId').get();
+
+            expect(connected).toBe(true);
```

Comment:
What's the idea here? 🤔  On line 68 we set the value to `true` ourselves..

Reply:
ups :D that should have been the callback